src/dotnet/UI.Blazor/Components/LayoutTypeProvider/LayoutType.cs

```diff
@@ -0,0 +1,3 @@
+namespace ActualChat.UI.Blazor.Components;
+
+public enum LayoutType { Desktop, Mobile  }
```

Comment:
What do you think about doing this a bit differently:
- `LayoutType` -> `record ScreenInfo` with property `ScreenSizeKind` (enum of `Unknown`, `Small`, `Medium`, `Large`, `ExtraLarge`, `ExtraLarge2`) -  https://tailwindcss.com/docs/responsive-design
- `IsDesktop` and `IsMobile` will be calculated based on above properties.
- Rename everything to `ScreenInfoXxx`.

Reply:
I kinda against `Layout` because it is a bit confusing - there is a concept of Layout in Blazor, which is different to this.

Reply:
Thought a bit more about this - maybe `enum ScreenSize` (no flags) with a bunch of extension methods is actually a better option, assuming:
- We set it to max available screen size (e.g. `ScreenSize.Medium`) & use just even numbers
- `Unknown` should be mapped to a value preceeding default screen size (e.g. `Medium`) & use the odd number
- `IsDesktop()`, etc. become extension methods.

Reply:
We can also implement `ScreenSize.IsMediumOrLarger()`, etc.

Reply:
+ Technically we can construct it as flags too - e.g. to simplify all these checks, we just don't have to use actual `[Flags]` attribute there.